dspace/config/dspace.cfg

```diff
@@ -102,6 +102,13 @@ db.maxidle = -1
 # pool.
 #db.jndi = jdbc/dspace
 
+# Whether or not to allow for an entire 'clean' of the DSpace database.
+# Enabling it would allow your database owner to destroy all DSpace data, tables, etc
+# by simply running 'dspace database clean' from commandline.
+# WARNING: NEVER ENABLE IN PRODUCTION. This tool is for development/testing only.
+# (Defaults to 'true', i.e. disabled)
+# db.cleanDisabled = true
```

Comment:
I see what you mean. The comments are a bit unclear.  We could also rename the config, but I chose to name it `db.cleanDisabled` as that's the same name Flyway gives it: https://github.com/DSpace/DSpace/pull/1468/files#diff-1cc848a50604fc2efb604de1cea81c70R258